src/go/configgenerator/filterconfig/update_proto_descriptor_test.go

```diff
@@ -1,348 +0,0 @@
-// Copyright 2021 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package filterconfig
-
-import (
-	"strings"
-	"testing"
-
-	"github.com/GoogleCloudPlatform/esp-v2/tests/utils"
-	descpb "github.com/golang/protobuf/protoc-gen-go/descriptor"
-
-	confpb "google.golang.org/genproto/googleapis/api/serviceconfig"
-	"google.golang.org/protobuf/encoding/prototext"
-	"google.golang.org/protobuf/proto"
-)
-
-func TestUpdateProtoDescriptor(t *testing.T) {
```

Comment:
where is this test?

Reply:
Moved to src/go/configgenerator/filtergen/grpc_transcoder_test.go